src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -301,7 +301,7 @@ export const ConversationPaginationContainer = createPaginationContainer(
         unread
         orderConnection(
           first: 10
-          states: [APPROVED, FULFILLED, SUBMITTED, REFUNDED, CANCELED]
+          states: [APPROVED, FULFILLED, SUBMITTED]
           participantType: BUYER
```

Comment:
we have some cases where we also query from the seller, for example when the seller sends you a counteroffer. In this case(Buyer only) the `lastOffer` won't reflect the real offer sent back by the Partner. [reference](https://github.com/artsy/eigen/blob/c0f23dff4b92c8a1f10f13500b87f1de2859044a/src/lib/Scenes/Inbox/Components/Conversations/OrderInformation.tsx#L31) also the order page contains the same logic as this reference